src/charm.py

```diff
@@ -263,5 +263,20 @@ def ip_is_valid(ip_address: str) -> bool:
         return False
 
 
+def ip_in_cidr_format_is_valid(ip_address: str) -> bool:
+    """Check whether given IP config is in CIDR format and valid.
+
+    Args:
+        ip_address (str): IP address in CIDR format
+
+    Returns:
+        bool: True if given IP address is valid
+    """
+    if "/" not in ip_address:
+        logger.warning(f"The IP address: {ip_address} is expected in CIDR format.")
```

Comment:
I think logging as error would be more appropriate than warning since we expect things to break at this point.

Reply:
Done